grazer/src/main/java/edu/asu/diging/grazer/core/domain/IFileTransformation.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.grazer.core.domain;
+
+import java.util.Date;
+
+/**
+ * This class stores all the details about the transformation files 
+ * i.e their IDs, labels, descriptions, file names, data uploaded, and uploader.
+ * 
+ * @author mshah18
+ *
+ */
+public interface IFileTransformation {
```

Comment:
is this representation one file or multiple?

Reply:
It is one pattern file and one transformation file

Reply:
IFile contains the actual transformation and pattern file data in bytes, and IFileTransformation contains the label, uploader, description etc of those transformation and pattern files. Both the files have the same label, id etc

Reply:
I think you need better names for your classes... ;)